sdk/src/storage/storageManager.ts

```diff
@@ -3,7 +3,7 @@
  * @description This module is responsible for managing JWT tokens.
  */
 
-const DEFAULT_COOKIE_EXPIRATION_YEARS = 10; // Default expiration for cookies is 10 years if not specified.
+const DEFAULT_COOKIE_EXPIRATION_YEARS = 10;
```

Comment:
I think we had chatted with @neildevas & @Asfiroth to lower this?

Reply:
now the life time is 2 weeks, I updated the branch with the recent changes